src/domain/transfer/index.js

```diff
@@ -75,7 +75,7 @@ const prepare = async (headers, message) => {
         }
       }
     }
-    const topicConfig = Utility.createGeneralTopicConf(TRANSFER, PREPARE, null, message.transferId)
+    const topicConfig = Utility.createGeneralTopicConf(TRANSFER, PREPARE, 0, null, message.transferId)
```

Comment:
This should be null. If you make it 0, it will ALWAYS go to partition 0. We want the partition to be assigned by the message Id (i.e. the transferId).

Reply:
fixed